buildenv/jenkins/JenkinsfileBase

```diff
@@ -773,7 +775,21 @@ def addGrinderLink() {
 def run_parallel_tests() {
 	if (params.PARALLEL && params.PARALLEL != "None") {
 		stage ("Parallel Tests") {
-			parallel parallel_tests
+			node{
```

Comment:
Please do not wrap `parallel parallel_tests` in node. We do not want to waste a machine to wait for parallel runs.